sources/platform/actors/publishing/monetize.mdx

```diff
@@ -9,16 +9,17 @@ sidebar_position: 2
 
 ---
 
-When publishing your Actor on the Apify platform, you have the choice to make it a **Paid Actor** and get paid by the users benefiting from your tool.
+Apify Store allows you to monetize your web scraping and automation projects by publishing them as Paid Actors. This guide explains the available pricing models and how to get started.
 
 ## Pricing models
 
-All Actors in [Apify Store](https://apify.com/store) fall into one of these three pricing models:
+Actors in the Apify Store can be published under one of the following pricing models:
 
-1. **free** - you can run the Actor freely, and you only pay for platform usage the Actor generates.
-2. **rental** - same as free, but in order to be able to run the Actor after the trial period, you need to rent it from the developer and pay a flat monthly fee on the top of the platform usage that the Actor generates.
-3. **paid per result** - you do not pay for the platform usage that the Actor generates but only for the results it produces.
+1. **Free**: Userc can run the Actor without any additional charges beyond the platform usage costs generated by the Actor
+2. **Rental**: Similar to the free model, users pay for the platform usage costs. However, after a trial period, they need to pay a flat monthly fee to the develop to continue using the Actor
+3. **Paid per Result**: Users don't pay for the platform usage costs. Instead, they pay the developer based on the number of results produced by the Actor.
+4. **Paid per Event**: Users don't pay for the platform usage cost the Actor generates. Instead, they pay based on specific events defined by the developer, such as generating a single result or initiating the Actor.
```

Comment:
Same as in the other PR, this is not yet available to general creator public, so maybe we can mark it such?